arcgis-ios-sdk-samples/Augmented reality/Collect data in AR/README.md

```diff
@@ -51,7 +51,7 @@ This sample only uses the `arScreenToLocation` approach, as it is the only way t
 
 Note that unlike other scene samples, a basemap isn't shown most of the time, because the real world provides the context. Only while calibrating is the basemap displayed at 50% opacity, to give the user a visual reference to compare to.
 
-**Real-scale AR** is one of three main patterns for working with geographic information in augmented reality. See the topic [Display scenes in augmented reality](https://developers.arcgis.com/ios/scenes-3d/display-scenes-in-augmented-reality/) in the iOS Guide for more information.
+**World-scale AR** is one of three main patterns for working with geographic information in augmented reality. See the topic [Display scenes in augmented reality](https://developers.arcgis.com/ios/scenes-3d/display-scenes-in-augmented-reality/) in the iOS Guide for more information.
```

Comment:
In `Maps/Apply scheduled updates to preplanned map area/README.md` it uses _in the *ArcGIS Developers* guide_; the same terminology should be used here.

```suggestion
**World-scale AR** is one of three main patterns for working with geographic information in augmented reality. See the topic [Display scenes in augmented reality](https://developers.arcgis.com/ios/scenes-3d/display-scenes-in-augmented-reality/) in the *ArcGIS Developers* guide for more information.
```